### DISCUSSION TITLE
Why was plot removed from the egui.rs demo?

### DISCUSSION THREAD

**Discussion by steckes** [emilk/egui#4995] [General]
I love showing people how amazing the plot touchpad controls are on egui.rs.
When I checked it today again the Plot widget is removed from the widget library and from the demo menu. Any plans on bringing this back and is there a reason why this disappeared?

**Comment by YgorSouza** [comment]
egui_plot was moved to a separate repo: <https://github.com/emilk/egui_plot>.

Reasoning in <https://github.com/emilk/egui/issues/4705>.

Since the plot code is no longer here, the demo cannot use it, or that would create a dependency loop. There is a new demo just for the plot at <https://emilk.github.io/egui_plot/>.

**Comment by steckes** [comment]
Perfect thank you for the hint, the standalone demo is more than enough :)
